build.gradle

```diff
@@ -251,11 +250,10 @@ dependencies {
      testCompile 'junit:junit:4.12'

```

Comment:
yeah I was just doing the few near my cursor. It's really a pretty un-necessary change at the moment but I hate having hardcoded version numbers